e2e-tests/controller-spark/controller_spark_sql_validation.sh

```diff
@@ -344,12 +347,60 @@ function validate_updated_resource() {
   fi
 }
 
+#################################################
+# Function that counts resources in  FHIR server and compares output to what is 
+#  in the source FHIR server
+# Globals:
+#   HOME_PATH
+#   SINK_FHIR_SERVER_URL
+#   TOTAL_TEST_PATIENTS
+#   TOTAL_TEST_ENCOUNTERS
+#   TOTAL_TEST_OBS
+#################################################
+function test_fhir_sink() {
+  local query_param="?_summary=count"
+
+  local runMode=$1
+
+  print_message "Finding number of patients, encounters and obs in FHIR server"
+
+  curl -L -X GET -u hapi:hapi --connect-timeout 5 --max-time 20 \
+    "${SINK_FHIR_SERVER_URL}/fhir/Patient${query_param}" 2>/dev/null >>"${HOME_PATH}/${PARQUET_SUBDIR}/patients-sink.json"
```

Comment:
nit: Please break long lines ([style guide](https://google.github.io/styleguide/shellguide.html#s5.2-line-length-and-long-strings)); here and everywhere else.

Reply:
done